### PR TITLE
[ADD] hr_holidays_team_manager

### PR DESCRIPTION
**Pull request by Murtaza-OSI**

### CODE REVIEW

**Comment by SMaciasOSI** [comment]
@max3903

**Review by max3903 — APPROVED** [review]

File: hr_holidays_team_manager/models/hr_employee.py [hunk 1]
```diff
@@ -47,7 +51,11 @@ def name_search(self, name="", args=None, operator="ilike", limit=100):
                     employee_id = self.env.user.with_context(by_pass=True).employee_ids
```
Comment (max3903) [inline hr_holidays_team_manager/models/hr_employee.py]:
```suggestion
```

**Comment by max3903** [comment]
@Murtaza-OSI Can you squash your commits?

**Pull request merged**